### DISCUSSION TITLE
What are the different pricing models offered by ChistaDATA Cloud?

### DISCUSSION THREAD

**Discussion by emoido** [ChistaDATA/Cloud#3] [❓FAQ] [answered]
What are the different pricing models offered by ChistaDATA Cloud?

**Comment by TheraniAA** [comment, marked as answer]
The pricing of our services is determined by the subscription tier chosen, with three options currently available: Free Tier, Startup, Enterprise Tier, and Critical Business Tier. For more details visit - https://chistadata.com/chistadata-cloud/
